test/toml_test.rb

```diff
@@ -1,7 +1,9 @@
 require_relative 'helper'
 require_relative 'toml_examples'
+require 'json'
+require 'pp'
```

Comment:
i forgot to remove it i was trying to have a better compare view of the json and toml in case they differ but pp doesn't seem to improve much